cli/cmd/configure_monitoring.go

```diff
@@ -54,7 +55,7 @@ keptn configure monitoring datadog --project=PROJECTNAME --service=SERVICENAME
 		return nil
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-		if args[0] == "prometheus" || args[0] == "datadog" {
+		if args[0] == "prometheus" || args[0] == "datadog" || args[0] == "sumologic" {
```

Comment:
this command does not fail when I make a typo error, for example:

`keptn configure monitoring sumollogic --project=PROJECTNAME --service=SERVICENAME`

I would suggest to negate the condition, something like:

if args[0] != "prometheus" && args[0] != "datadog" && args[0] != "sumologic" { return err }

and afterwards test if the --project and --service flags are present

Reply:
also, pls add tests for these use-cases

Reply:
Thank you for the comment Ondrej. I will fix this. I am facing an issue in the main sumologic-service. I will get to this once I fix the issue there (I want to fix the issue in sumologic-service before we merge this PR). I will mention you here once I have updated this PR.